gestalt-android/build.gradle

```diff
@@ -51,3 +57,61 @@ dependencies {
     androidTestImplementation 'com.android.support.test:runner:1.0.2'
     androidTestImplementation 'com.android.support.test.espresso:espresso-core:3.0.2'
 }
+
+
+// Artifactory publishing details
+buildscript {
```

Comment:
On the other hand repeating this big section is a little awkward.

Reply:
Yep! Will handle.